python_modules/libraries/dagster-databricks/dagster_databricks/unity_catalog.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+
+from typing import Optional, Sequence
+
+from dagster import AssetKey, AssetsDefinition, TableColumn, TableSchema
+from dagster._core.definitions.asset_spec import AssetSpec
+from dagster._core.definitions.external_asset import external_asset_from_spec
+from dagster._annotations import preview
+from databricks.sdk import WorkspaceClient
+
+import requests
+
+@preview
+def unity_catalog_assets(
```

Comment:
I think this function could have an associated test, just monkeypatch or mock the returns from the API/SDK.